src/components/Application-Header/favorite-list/FavoriteList.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+
+import { observer, inject } from 'mobx-react';
+
+import { withStyles, WithStyles } from '@material-ui/core';
+import IconButton from '@material-ui/core/IconButton';
+import StarIcon from '@material-ui/icons/Star';
+import Popover from '@material-ui/core/Popover';
+import Divider from '@material-ui/core/Divider';
+
+import FavoriteItem from './FavoriteItem';
+import { JobsStore } from '../../../store/JobsStore';
+import styles from './FavoriteListStyles';
+
+interface FavoriteListProps extends WithStyles<typeof styles> {
+    jobsStore?: JobsStore;
+}
+
+const FavoriteList: React.FC<FavoriteListProps> = (props): JSX.Element => {
+    const { classes } = props;
+    const jobsStore = props.jobsStore!;
+
+    const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
```

Comment:
I see that your'e only using the anchor to determine whether to display the popover or not, IMO, it's a bit vague and boolean named `isPopoverOpen` would be great.
But it's up to you.

Reply:
it's not only for open/close flag, it holds the location in which the popover will open (based on the mouse click position I think).